src/main/java/com/firstwebapp/practice/spring/SongController.java

```diff
@@ -0,0 +1,28 @@
+package com.firstwebapp.practice.spring;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.servlet.view.RedirectView;
+
+    @Controller
+    public class SongController {
+
+        @Autowired
+        private SongRepository songRepo;
+
+        @ResponseBody
+        @RequestMapping(value = "/songs", method = RequestMethod.GET)

```

Comment:
You have a lot of code here, but absolutely none of it is working: when I start up your app, I can't see a single route that's functional.